test/verifiedTests/testSolvers/NLPscripts/NLP_objFunction.m

```diff
@@ -0,0 +1,11 @@
+function f = NLP_objFunction(x, Prob)
+% f = NLP_objFunction(x, Prob)
+% From tomlab quickguide
```

Comment:
This problem is called the Rosenbrock problem. We should add this in the doctring.